core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java

```diff
@@ -151,7 +152,6 @@ public void execute(Event<UIDocumentAutoVersionForm> event) throws Exception {
             PasteManageComponent.pasteByCut(autoVersionComponent.getCurrentClipboard(), uiExplorer, destSession, autoVersionComponent.getCurrentClipboard().getWorkspace(),
                     sourceNode.getPath(), destPath, WCMCoreUtils.getService(ActionServiceContainer.class), false,false, false);
           }else {
-            // nt:folder Does not support SNS
```

Comment:
Keep also this comment

Reply:
I added the comment **// nt:folder Does not support SNS** to the code